test/test_um2netcdf.py

```diff
@@ -186,3 +186,34 @@ def test_check_pressure_level_masking_missing_heaviside_t(ta_plev_cube):
 
     assert need_heaviside_t
     assert heaviside_t is None
+
+
+# cube filtering tests
+# use wrap results in tuples to capture generator output in sequence
+
+def test_cube_filtering_mutually_exclusive():
+    include = [30201]
+    exclude = [30293]
+    cubes = (ua_plev_cube, heaviside_uv_cube)
```

Comment:
Just checking - does this test need to request the  `ua_plev_cube` and `heaviside_uv_cube` fixtures to work properly?

Reply:
Or actually does it not matter too much here as the `filtered_cubes` function raises the value error before doing anything with the cubes.

Reply:
I've fixed it for completeness.